### DISCUSSION TITLE
make it easier to navigate across columns by mouse

### DISCUSSION THREAD

**Discussion by KiaraGrouwstra** [YaLTeR/niri#165] [Ideas]
in paperwm, when focusing/moving a window it would tend to ensure not just that the window pans/stays into view, but that in fact one could see just beyond its gap, as might be seen in their [screenshot](https://github.com/paperwm/media/blob/master/tiling.png).
this is relevant when navigating by mouse, as having such a window edge in view would allow one to click it and focus that, in turn panning the camera to bring the next edge into view.
it would be nice if niri would facilitate this as well, even if optional.

**Comment by YaLTeR** [comment]
You can replicate this with horizontal struts:
```
layout {
    struts {
        left 16
        right 16
    }
}
```

**Reply by KiaraGrouwstra** [reply]
nice, thanks!
